hips/utils/healpix.py

```diff
@@ -50,7 +53,49 @@ def boundaries(nside: int, pix: int, nest: bool=True) -> tuple:
         [( 264.375, -24.62431835), ( 258.75 , -30.        ),
          ( 264.375, -35.68533471), ( 270.   , -30.        )]>
     """
-
     boundary_coords = hp.boundaries(nside, pix, nest=nest)
     theta, phi = hp.vec2ang(np.transpose(boundary_coords))
     return theta, phi
+
+def compute_healpix_pixel_indices(wcs_geometry: WCSGeometry, nside: int) -> np.ndarray:
+    """Returns an array containing HEALPix pixels corresponding to disk regions.
+
+    This function calls `healpy.pixelfunc.ang2vec`, `healpy.query_disc`, and
+    `astropy.coordinates.angle_utilities.angular_separation` to compute
+    the HEALPix pixel indices, which will be used in tile drawing.
+
+    Parameters
+    ----------
+    wcs_geometry : WCSGeometry
+        Container for WCS object and image shape
+    nside : int
+        The nside of the HEALPix map
+
+    Returns
+    -------
+    pixels : `numpy.ndarray`
+        HEALPix pixel numbers
+
+    Examples
+    --------
+    ::
+
+        >>> import healpy as hp
+        >>> from hips.utils import WCSGeometry
+        >>> from astropy.coordinates import SkyCoord
+        >>> from hips.utils import compute_healpix_pixel_indices
+        >>> order = 3
+        >>> nside = hp.order2nside(order)
+        >>> skycoord = SkyCoord(10, 20, unit="deg")
+        >>> wcs_geometry = WCSGeometry.create(skydir=skycoord, shape=(10, 20), coordsys='CEL', projection='AIT', cdelt=1.0, crpix=(1., 1.))
+        >>> compute_healpix_pixel_indices(wcs_geometry, nside)
+        [ 84 111 112 113 142 143 144 145 146 174 175 176 177 178 206 207 208 209
+        210 238 239 240 241 270 271 272 273 274 302 303 304 305 334 335 336 337
+        367 368 399]
+    """
+    y_center, x_center = wcs_geometry.shape[0] // 2, wcs_geometry.shape[1] // 2
```

Comment:
Actually, I think here it should be `/` (float division) and not `//` (integer devision, which I think I suggested before).